### DISCUSSION TITLE
crew_controller_slurm seems to work

### DISCUSSION THREAD

**Discussion by kendonB** [wlandau/crew.cluster#21] [General]
I was very pleased to get targets working with crew on my SLURM cluster (NeSI).

I was particularly surprised that it figured out which account to use and which modules to load (which may be a feature of NeSI rather than crew).

Nice work!

**Comment by wlandau** [comment]
Thanks for trying it out!

{crew}/{crew.cluster} does not get involved in specialized configuration settings like accounts, credentials, or modules, but if you ever need to, you can set them using the script_lines argument of the controller.
